commands/core/install.go

```diff
@@ -26,18 +26,37 @@ import (
 )
 
 // PlatformInstall FIXMEDOC
-func PlatformInstall(ctx context.Context, req *rpc.PlatformInstallRequest,
-	downloadCB rpc.DownloadProgressCB, taskCB rpc.TaskProgressCB) (*rpc.PlatformInstallResponse, error) {
+func PlatformInstall(
+	ctx context.Context,
+	req *rpc.PlatformInstallRequest,
+	downloadCB rpc.DownloadProgressCB,
+	taskCB rpc.TaskProgressCB,
+) (*rpc.PlatformInstallResponse, error) {
+	if err := platformInstall(ctx, req, downloadCB, taskCB); err != nil {
+		return nil, err
+	}
+	if err := commands.Init(&rpc.InitRequest{Instance: req.Instance}, nil); err != nil {
+		return nil, err
+	}
+	return &rpc.PlatformInstallResponse{}, nil
+}
 
+// platformInstall is the implementation of platform install
+func platformInstall(
```

Comment:
use a nested function, this way we could remove parameter duplication and could use the defer statement